python/python_fastapi_server/routers/edr.py

```diff
@@ -420,11 +471,16 @@ def get_params_for_collection(edr_collection: str) -> dict[str, Parameter]:
         else:
             label = param_el["name"]
 
+        symbol = Symbol(value=param_el["unit"], type=SYMBOL_TYPE_URL)
+        param_id = param_el["name"]
+        # If standard name was configured, use that instead with a vocabulary
+        if "standard_name" in param_el:
+            param_id = VOCAB_ENDPOINT_URL + param_el["standard_name"]
         param = Parameter(
```

Comment:
This is the `Parameter` object that is used in the metadata in the `/collections` endpoint. But if you do a data query, you also get a `Parameter` object back which has a slightly different type (long story...). This is constructed on line 1056 of edr.py. I think we want to update that as well?

Reply:
Ah yes!